spring-native-configuration/src/main/java/org/springframework/kafka/annotation/KafkaHints.java

```diff
@@ -69,7 +69,7 @@
 import io.confluent.kafka.serializers.KafkaAvroSerializer;
 import io.confluent.kafka.serializers.subject.TopicNameStrategy;
 
-@NativeHint(trigger=KafkaListenerConfigurationSelector.class,
+@NativeHint(trigger=KafkaListener.class,
```

Comment:
I think `KafkaListener` will be evaluated via a classpath check, could we use for example`KafkaListenerAnnotationBeanPostProcessor` instead which will be enabled only if such bean is active?